src/controllers/admin.js

```diff
@@ -0,0 +1,26 @@
+/* eslint-disable require-jsdoc */
+import Response from '../helpers/sendResponse';
+import models from '../database/models';
+import managers from '../helpers/manager';
+import roles from '../helpers/roles';
+
+export default class AdminController {
+  static async updateRole(req, res) {
+    try {
+      const { newRole } = req.body;
+      const { email } = req.params;
+      const user = await models.User.findOne({ where: { email } });
```

Comment:
As we agreed, better do not use models in controllers, instead, create a separate file in a folder called queries or services and call it in the controller.

Reply:
@harerakalex it have been corrected. you can check and let me know if I missed something.